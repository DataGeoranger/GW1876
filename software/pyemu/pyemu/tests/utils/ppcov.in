utils\points1_test.dat
0.0
utils\struct_test.dat
struct3
utils\ppcov.struct3.out

