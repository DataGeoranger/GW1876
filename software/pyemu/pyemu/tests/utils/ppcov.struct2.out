     3     3     1
   1.000000      0.6939921      0.5222003    
  0.6939921       1.000000      0.4274848    
  0.5222003      0.4274848       1.000000    
* row and column names
 hkpp1
 hkpp2
 hkpp3
